docs/2023-06-08-meeting-transcript.md

```diff
@@ -0,0 +1,881 @@
+
+Minecraft needs, like, the Internet to login into, to play on servers, and another thing we knida need is filesystem access to the instance folder, the `.minecraft` and its recursive descedents. It bascially doesn't need any other thing, right? After that. 
+
+The problem here, or the interesting theory here to lock down filesystem access, but the problem is that, we breifly kinda looked into this, and it's basically - the state of things is that, Windows has like, no facility for this, basically, at all, and Windows is where like, 95% of the users are. 
+
+The agenda has some light discussion for macOS and Linux, and people are kinda building prototypes on that, which is cool, we can do that where it's possible, but I'd urge people have to focus on Winodws cause that's where all the players are.  
+
+Just the state of things there, doesn't look that great. 
+
+So, I don't know if there is actually that much to discuss, but since it was brought up multiple times during the whole incident, I thought to bring it up here again. 
+
+One of the other thing, that I'm just gonna mention out of the gate, is that JVM-level sandboxing is just, not gonna work. Besides the fact that it is getting deprecated and removed in a few years, it's just broken, so don't suggest that. 
+
+Alright, so open the floor for discussion. 
+
+Emi: Yeah, we have a couple of people from the Prism Launcher team that would like to speak on this topic, so I am gonna invite them in to have their perspective on this issue. 
+
+ZekeZ: Good. Hi. How's everyone going today?
+
+timoreo: Hello hello.
+
+ZekeZ: Alright, um. Do you want to start this timoreo? You're the main, one of the main developers behind the actual launcher. 
+
+timoreo: Yeah. So I'll to start. So, sandboxing, first off, the first point: executing untruest will never be safe, no matter how much sandboxing you put, that's just a fact. [ZekeZ: um-hum.] You can never fully protect. 
+
+This is kind of, sandboxing is the last resort, just to limit the impact. 
+
+In fact, we can do a bits of sandboxing because we have a distribution of Flatpak, which is sandboxed. 
+
+It may seem a bit like a niche part, except that what's important is that Steam Deck users, which has actually a huge majority of their users, is forced to use Flatpak.
+
+What Flatpak does is essentially restrict a lot of access, mainly file systems.  
+
+So in this case, for this specific malware Fractureizer, it wouldn't have been able to steal information from the browser and all of that, or set up starting some malicious code that booted all of that.
+
+ZekeZ: And yeah, in this case, we found it was ineffective, the malware, I believe, against the Flatpak. Is that correct?
+
+timoreo: Yes. I mean, generally, the malware wasn't that well coded for Linux.
+
+ZekeZ: This specific one, at least.
+
+timoreo: This specific one was entirely broken on Linux. 
+
+But the problem is kind of, you can restrict more with Flatpaks, but you restrict ???. 
```

Comment:
I think this is just them tripping over their words, and the next line is what they intended to say.

Reply:
Now I would assume the same. Fixed by replacing with "but... for example, you could restrict microphone, ...".